src/features/content-answers/InlineError.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import BoxAi from '../../icon/line/BoxAi';
+import Media from '../../components/media';
+import TriangleAlert16 from '../../icon/line/TriangleAlert16';
+// @ts-ignore: no ts definition
+// eslint-disable-next-line import/named
+import { ElementsXhrError } from '../../common/types/api';
+
+import messages from './messages';
+
+import './InlineError.scss';
+
+type Props = {
+    error: ElementsXhrError;
+};
+
+const InlineError = ({ error }: Props) => {
```

Comment:
do we have to pass in the `error`? you're already controlling when to render component in the parent component

Reply:
removed error prop, but in the future we could possibly pass error and make the inlineError component show different error messages based on the error prop